mypyc/test-data/irbuild-statements.test

```diff
@@ -607,6 +607,91 @@ L5:
 L6:
     return 1
 
+[case testStarUnpack]
+from typing import Any, List, Iterator
+
+it: Iterator = iter(['x', 'y', 'z1', 'z2', 'z3', 'u', 'w'])
+
+def f(a: Any) -> None:
+    a.x, a.y, *a.z, a.u, a.w = it
+[out]
+def f(a):
+    a :: object
+    r0 :: dict
+    r1 :: str
+    r2, r3, r4 :: object
+    r5 :: bool
+    r6 :: object
+    r7 :: bool
+    r8 :: str
+    r9 :: int32
+    r10 :: bit
+    r11 :: str
+    r12 :: int32
+    r13 :: bit
+    r14 :: list
+    r15 :: ptr
+    r16 :: int64
```

Comment:
I tested this on lots different test cases from both this branch and using master. Each time it generates a register whose size is platform dependent (int64 for 64 bit platform and int32 otherwise)  This is actually decided by `c_pyssize_t_rprimitive` inside `ll_builder.py.builtin_len()`. Not sure how to write a system independent irbuild test case for it :thinking:

Reply:
ummm that's due to https://github.com/mypyc/mypyc/issues/776. A quick fix to pass the CI is to change all `int64` in your generated IR output into `native_int`.

Reply:
Thanks, it works now :)